esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -2990,8 +2993,10 @@ int WUSchedule::run()
         ERRLOG("Unknown exception in WsWorkunits Schedule::run");
     }
 
-    if (m_container)
-        m_container->exitESP();
+    //Rodrigo: do we need to do this anyway?? why not attempt to iterate until dali is reachable again??
+    //but anyway do we want to notify this thread that we're attached/detached?
```

Comment:
Needs investigating.

Reply:
@wangkx an accessible DALI will no longer be a requirement for ESP Processes, do you see any issue stemming from removing line 2993-2994 ?

Reply:
@rpastrana if those 2 lines are removed, an alternative solution has to be added to restart the WUSchedule loop whenever dali is accessible again.